views/analysis.py

```diff
@@ -200,13 +200,16 @@ def get_context_data(self, **kwargs):
         sections = self.options_panel_dict()
         try:
             analysis = Analysis.objects.get(id=self.kwargs.get('pk'))
+            is_analysis_running = True
```

Comment:
do not use this.
Use ```analysis.get_task_state()``` to determine analysis status.
Reason:
We are using celery to run analysis. To get the state of the task reliably, use ```get_task_state()```. It will return celery task state: `PENDING`, `FAILURE`, `SUCCESS`. Use this state and adapt for your view.
For example, `PENDING` means the task is created but not yet completed, so it is synonymous with your `is_analysis_running` variable. `PENDING` means it is currently being run (or waiting to be run).